### ISSUE TITLE
ie8 compatibility bug

### ISSUE THREAD

**Issue by wizraopeize** [jakubpawlowicz/clean-css#528]
```
.sys-msg {
    border: 1px solid #cccccc\0;
}
```
After I using `cleancss -c 'ie8' test.css `, this is the result.
```
.sys-msg{border:1px solid}
```

**Comment by jakubpawlowicz** [comment]
It's a bug indeed.

**Comment by jakubpawlowicz** [comment]
Will be released as part of 3.2 in the coming days.
